internal/app/machined/pkg/controllers/network/route_spec.go

```diff
@@ -203,7 +203,9 @@ func (ctrl *RouteSpecController) syncRoute(ctx context.Context, r controller.Run
 				existing.Protocol == uint8(route.TypedSpec().Protocol) &&
 				existing.Attributes.OutIface == linkIndex && existing.Attributes.Priority == route.TypedSpec().Priority &&
 				(value.IsZero(route.TypedSpec().Source) ||
-					existing.Attributes.Src.Equal(route.TypedSpec().Source.AsSlice())) {
+					existing.Attributes.Src.Equal(route.TypedSpec().Source.AsSlice())) &&
```

Comment:
it seems there are other route parameters where we don't check if the spec matches completely, so updates are just no-op, until a reboot

Reply:
we could probably simplify this a bit with something like this before `if`:

```go
var existingMTU uint32

if existing.Attributes.Metrics != nil {
  existingMTU = existing.Attributes.Metrics.MTU
}


...

|| existingMTU == route.TypedSpec().MTU
```